src/main/drivers/display_ug2864hsweg01.c

```diff
@@ -215,6 +216,24 @@ void i2c_OLED_clear_display_quick(busDevice_t *bus)
     }
 }
 
+void i2c_OLED_set_horizontal_adressing_mode(busDevice_t *bus)
```

Comment:
`i2c_OLED_set_adressing_mode(busDevice_t *bus, bool vertical)` ? Or use enum for addressing modes ...